cmd/proxy/Dockerfile

```diff
@@ -1,8 +1,12 @@
 # Links to compare against to ensure we have all VCS's setup in this build
 # https://github.com/docker-library/buildpack-deps/blob/1845b3f918f69b4c97912b0d4d68a5658458e84f/stretch/scm/Dockerfile
 # https://github.com/golang/go/blob/f082dbfd4f23b0c95ee1de5c2b091dad2ff6d930/src/cmd/go/internal/get/vcs.go#L90
-
-FROM golang:1.13-alpine AS builder
+#
+# You can override the Go version used to build the image.
+# See project Makefile if using make.
+# See docker --build-arg if building directly.
```

Comment:
@arschles Here's my first take on adding a comment regarding the use of the `GOLANG_VERSION` override.

Please review and let me know if you have any comments/suggestions.

-DF